README.md

```diff
@@ -0,0 +1,27 @@
+# GFF NF1 organoid image profiling
```

Comment:
Maybe this comes later in the future, but could describe NF1 more or provide links, and other background info about the organoid experiment data.

Reply:
I think most of this comes later. NF1 links could be helpful and the only other experiment info that could be included is the microscope which I don't remember off the top of my head.

I think in further PRs this README will be updated more to include things as we progress.